spec/abstractops.html

```diff
@@ -201,14 +201,15 @@ <h1>
     <dl class="header">
       <dt>description</dt>
       <dd>
-        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it represents a finite number greater than or equal to 1, and returns that value truncated to an integer.
+        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it represents a number in the inclusive range 1 through one billion, and returns that value truncated to an integer.
       </dd>
     </dl>
     <emu-alg>
       1. Let _increment_ be ? GetOption(_normalizedOptions_, *"roundingIncrement"*, *"number"*, *undefined*, *1*<sub>𝔽</sub>).
       1. If _increment_ is not finite, throw a *RangeError* exception.
-      1. If _increment_ &lt; *1*<sub>𝔽</sub>, throw a *RangeError* exception.
-      1. Return truncate(ℝ(_increment_)).
+      1. Let _integerIncrement_ be truncate(ℝ(_increment_)).
+      1. If _integerIncrement_ &lt; 1 or _integerIncrement_ &gt; 10<sup>9</sup>, throw a *RangeError* exception.
+      1. Return _integerIncrement_.
```

Comment:
@gibson042 I thought you might be particularly interested to note this change since you did an earlier similar change to check the range for `fractionalSecondDigits` after truncating it; now that we have an upper limit, it matters here too.